### DISCUSSION TITLE
Same airtag shows up as multiple devices over time

### DISCUSSION THREAD

**Discussion by luke-jr** [seemoo-lab/AirGuard#106] [Q&A]
Ignoring an airtag is a temporary solution, but it soon reappears as a "new" airtag...

Is there any way to connect to it and read the serial number or something? (Maybe requires the private key?)

**Comment by larsdennert** [comment]
Unfortunately not. Only the airtag owner can sequence the Mac randomization to id their airtag. Seems apple devices are now randomizing their Mac address much more than once a day. Seems to happen in less than half an hour now.

**Comment by Sn0wfreezeDev** [comment]
AirTags change their Mac once a day. We cannot combine them to the same AirTag. This would only work if they would be monitored very closely, like you put your phone next to it and scan actively until it changes the Mac.
